ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.6.yaml

```diff
@@ -0,0 +1,80 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v0.16.0_knative-eventing-src:
+    name: knative-v0.16.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.16.0_knative-serving-src:
+    name: knative-v0.16.0
+    namespace: openshift
+    tag: knative-serving-src
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.16.0_knative-eventing-src:
+      as:
+      - registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-src
+    openshift_knative-v0.16.0_knative-serving-src:
+      as:
+      - registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-src
```

Comment:
Do we need these anymore? I think they're obsolete with us adding that to the build image?

Reply:
This is some automatic metadata that was added by automation CI tools. We define it manually here: https://github.com/openshift/release/pull/12234/files#diff-3cbc3dcd726d5c860a68e7956c097286R10
Anyway, I think this is required. This is how they get to the serverless-operator-src image, AFAIK.